samples/Mvc/Mvc.Client/Views/Shared/Home.cshtml

```diff
@@ -2,7 +2,7 @@
 
 <div class="jumbotron">
     @if (User?.Identity?.IsAuthenticated ?? false) {
-        <h1>Welcome, @User.Identity.Name</h1>
+        <h1>Hi, @Context.User.Identity.Name</h1>
```

Comment:
That's not something I changed recently: https://github.com/aspnet-contrib/AspNet.Security.OpenIdConnect.Server/blob/dev/samples/Mvc/Mvc.Client/Views/Shared/Home.cshtml :smile: